README.md

```diff
@@ -20,37 +20,56 @@ Installation
 ------------
 Steps to installation:
 
+* Install pre-requisites: Git, Python3, and lots of things for Pillow.
 * Clone the repository
 * Change into the cloned repository
-* Create a virtual environment (note this application uses python3)
+* Create a virtual environment
 * Activate the virtual environment
 * Install dependencies
 * Install Java 7+ (this is a requirement for Elasticsearch but the packages don't depend on any Java package)
 * Install Elastic search
+* Run database migrations
 * Run the development server
 
-On a Debian-based Linux distribution, your commands will probably look like this:
+These instructions assume Ubuntu Trusty 64.
+
 ```
+sudo apt-get install -y git-core
+sudo apt-get install -y libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python-tk # Requirements for pillow
```

Comment:
Looks like we avoid this by installing the pillow wheel. I'll update the README to explain how to do that.

Reply:
This needn't block merging this, so new issue https://github.com/OpenDataServices/grantnav/issues/406